src/sentry/issues/issue_occurrence.py

```diff
@@ -34,6 +34,7 @@ class IssueOccurrenceData(TypedDict):
     detection_time: float
     level: Optional[str]
     culprit: Optional[str]
+    transaction_duration: Optional[int]
```

Comment:
Hmm, why is this part of the occurrence as opposed to the event or something? This seems specific to detections that use transactions?